packages/client-sdk-nodejs/src/batchutils/batch-functions.ts

```diff
@@ -0,0 +1,145 @@
+import {CacheDelete, CacheGet, CacheSet} from '@gomomento/sdk-core';
+import {CacheClient} from '../cache-client';
+import {
+  BatchDeleteRequest,
+  BatchDeleteResponse,
+  BatchGetRequest,
+  BatchGetResponse,
+  BatchSetItem,
+  BatchSetRequest,
+  BatchSetResponse,
+  defaultMaxConcurrentRequests,
+} from './batch-props';
+import {range} from '@gomomento/sdk-core/dist/src/internal/utils';
+
+export {
+  BatchDeleteRequest,
+  BatchDeleteResponse,
+  BatchGetRequest,
+  BatchGetResponse,
+  BatchSetItem,
+  BatchSetRequest,
+  BatchSetResponse,
+  defaultMaxConcurrentRequests,
+} from './batch-props';
+
+export async function batchGet(
+  request: BatchGetRequest
```

Comment:
This Request-based pattern for our APIs is kind of unique to our golang SDK. For TS, I think you will want the signature to look something like this:

```
export async function batchGet(cacheClient: ICacheClient, cacheName: string, keys: Array<string | UInt8Array>, options?: BatchGetOptions);
```

where `string | UInt8Array` should just be whatever we normally accept for a `get`, and the `BatchOptions` interface would look something like this:

```
export interface BatchOptions {
   maxConcurrentRequests?: number
}
```

Same comment for the other 2 APIs.